Sign-in Alerts (#1215)
appwrite/website: src/partials/auth-security.md

Encourage passwords that are hard to guess by disallowing users to pick passwords that contain personal data.
Personal data includes the user's name, email, and phone number.

Disallowing personal data can be enabled in the Auth service's **Security** tab on the Appwrite Console.

# Session alerts {% #session-alerts %}

> Set up email alerts for your users so that whenever another session is created outside of their existing session using their user account information, such as their email address, they will be alerted to the new session.

Reviewer: > Set up email alerts

There's not much to set up. It's just enabling/disabling so we can just say enable.

> outside of their existing session using their user account information, such as their email address,

This is really long-winded. Best to be short and concise.

```suggestion
Enable email alerts for your users so that whenever another session is created for their account, they will be alerted to the new session.
```

> Users must have a valid email address to receive email notifications. This isn't the case when using [Magic URL](/docs/products/auth/magic-url) login, [Email OTP](/docs/products/auth/email-otp) login, or [OAuth2](/docs/products/auth/oauth2) login. To toggle session alerts, navigate to **Auth** > **Security** > Session Alerts.

Reviewer: > Users must have a valid email address to receive email notifications. This isn't the case when using

This makes it sound like Magic URL, Email OTP, and OAuth2 are not cases with valid emails.

> To toggle session alerts, navigate to **Auth** > **Security** > Session Alerts.
Comment

Let's move this to a separate paragraph because it's a different topic.

Encourage passwords that are hard to guess by disallowing users to pick passwords that contain personal data.
Personal data includes the user's name, email, and phone number.

Disallowing personal data can be enabled in the Auth service's **Security** tab on the Appwrite Console.

# Session alerts {% #session-alerts %}

Enable email alerts for your users so that whenever another session is created for their account, they will be alerted to the new session.

> Users must have a valid email address to receive email notifications. But you don't receive notifications when logging in using [Magic URL](/docs/products/auth/magic-url), [Email OTP](/docs/products/auth/email-otp), or [OAuth2](/docs/products/auth/oauth2). 

Reviewer: > Users must have a valid email address to receive email notifications. 

Let's actually leave this out because it's kind of self-explanatory

> But you don't receive notifications

Saying "but" like this sounds a little awkward/out of place. Maybe it would be helpful to expand this by explaining a bit. The goal of session alerts is to notify users if someone unauthorized logged into their account. We probably decided to not send session alerts for magic URL, email OTP, and OAuth2 because those are more likely to be legitimate auths since they must have access to these other systems rather than just a password.

> You won't receive notifications when logging in using [Magic URL](/docs/products/auth/magic-url), [Email OTP](/docs/products/auth/email-otp), or [OAuth2](/docs/products/auth/oauth2) since these authentication methods already verify user access to their systems, verifying the authentication's legitimacy.

Reviewer: Can we reword to avoid using both "verify" and "verifying"?

> To toggle session alerts, navigate to **Auth** > **Security** > Session Alerts.

Reviewer: Let's bold session alerts since [all text seen as-is in the UI should be bold](https://github.com/appwrite/website/blob/7a7f74f4c6acba1937505aa989f5626867e272e7/CONTENT.md?plain=1#L110) and make it sentence case so it matches the Console.

```suggestion
To toggle session alerts, navigate to **Auth** > **Security** > **Session alerts**.
```
